packages/typescriptlang-org/src/copy/en/community.ts

```diff
@@ -1,8 +1,8 @@
 export const comCopy = {
-  com_layout_title: "How to set up TypeScript",
+  com_layout_title: "How to set up TypeScript", // FIXME: Is this the right title for the community page?
```

Comment:
_might_ be used in the HTML title, but if not yeah, could be a bad C&P

Reply:
confirmed:
![Screen Shot 2020-07-30 at 5 23 05 PM](https://user-images.githubusercontent.com/49038/88976029-52385f00-d289-11ea-94b6-51ec6c66af0c.png)


It's wrong in English, I think it should probably just be "Connect with other TypeScripters online and offline"